src/libs/kata-types/src/config/default.rs

```diff
@@ -14,8 +14,14 @@ use lazy_static::lazy_static;
 lazy_static! {
     /// Default configuration file paths, vendor may extend the list
     pub static ref DEFAULT_RUNTIME_CONFIGURATIONS: Vec::<&'static str> = vec![
+        "/etc/kata-containers/configuration-dragonball.toml",
+        "/usr/share/defaults/kata-containers/configuration-dragonball.toml",
+        "/opt/kata/share/defaults/kata-containers/configuration-dragonball.toml",
+        "/opt/confindetial-containers/share/defaults/kata-containers/configuration-dragonball.toml",
```

Comment:
Is creating a soft link from `/../../configuration.toml` to `../../configuration-dragonball.toml` during CI environment builtup process or user installation process a better way?

Reply:
So, it depends a lot.  CI does that, but kata-deploy and in the payload we provide to Confidential Containers we don't do that.

The reason for that is because the default runtime and the default VMM are still the go one and QEMU.  So, IOW, if we just link `configuration-dragonball.toml` to `configuration.toml` we break the current behaviour and the current assumption of the users.

Reply:
Will this change cause go runtime to use `configuration-dragonball.toml` since it's the first option? (If go runtime and rust runtime are both installed in some cases? I think that is a situation in kata-deploy if I remember correctly)

Reply:
Yep, that's exactly the situation with kata-deploy, where we'll have both the go and the rust runtime "installed" at the same time.

The changes that I did were to avoid having any kind of conflict in that specific case (which is also the case for Confidential Containers).

If we just linked `configuration-dragonball.toml` toi `configuration.toml` the `kata` runtime class would pick the go runtime, to use with QEMU, but using the DB configuration, and that would simply explode. :-)

Is there something else that I could try to explain in a better way?

Reply:
By the way, http://jenkins.katacontainers.io/job/kata-containers-dragonball-tests-ubuntu-20.04-PR/372/console now has passed the point of the issues that I had before this PR. \o/

Reply:
ohhh I got it. I misunderstood that `DEFAULT_RUNTIME_CONFIGURATIONS ` will also be used in go-runtime. So I thought this will influence go runtiem cause they will also choose dragonball config as default. My mistake, sorry.

But if later CLH/QEMU introduce their support for runtime-rs, will this change here makes it hard for them to use their config? Maybe a FIXME or TODO comments leave here?

Reply:
> But if later CLH/QEMU introduce their support for runtime-rs, will this change here makes it hard for them to use their config?

That's a valid point, but I sincerely hope we are able to get rid of this by having https://github.com/kata-containers/kata-containers/issues/3961 implemented.

Reply:
I've updated the patch with a TODO comment.
Thanks for the review!

Reply:
Thanks!